### PR TITLE
Reflect storage location types

### PR DESCRIPTION
**Pull request by jtweed**
### What is this PR trying to achieve?

Be more explicit about where things are stored and on what kind of storage.

### Who is this change for?

Everyone who wants to know where things are.

### CODE REVIEW

File: docs/rfcs/002-archival_storage/README.md [hunk 1]
```diff
@@ -560,9 +584,33 @@ Response:
...
+  "archiveLocation": {
+    "type": "DigitalLocation",
+    "locationType": {
+      "type": "LocationType",
+      "id": "aws-s3-glacier",
+      "label": "AWS S3 - Glacier"
+    },
+    "url": "s3://bucketname.s3-eu-west-1.amazonaws.com/born_digital/GC253_1046-a2870a2d-5111-403f-b092-45c569ef9476"
+  },
+  "replicaLocation": {
```
Comment (mklander) [inline docs/rfcs/002-archival_storage/README.md]: can these be lists? `replicaLocations=[{...}]`

**Comment by kenoir** [comment]
See https://github.com/wellcometrust/platform/pull/2864.

The `s3://` urls here are unfortunately invalid - it requires a little bit of inferring from documentation, (see http://www.wryway.com/blog/aws-s3-url-styles/, https://docs.aws.amazon.com/general/latest/gr/rande.html#s3_region & https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingBucket.html).

Credit to @fractos for figuring this out.

File: docs/rfcs/002-archival_storage/README.md (hunk 1 above)
Comment (jtweed) [reply docs/rfcs/002-archival_storage/README.md]: Updating as per our conversation.

**Comment by jtweed** [comment]
@kenoir You missed a discussion in the office - we've changed to not use URLs at all.

**Comment by kenoir** [comment]
> @kenoir You missed a discussion in the office just now - we've changed to not use URLs at all.

pivot

**Pull request merged**